### ISSUE TITLE
Fix the license that GitHub shows

### ISSUE THREAD

**Issue by kyleneideck** [kyleneideck/BackgroundMusic#460]
The [LICENSE file](https://github.com/kyleneideck/BackgroundMusic/blob/master/LICENSE) is a symlink, so GitHub can't figure out what Background Music's license and just shows "view license" on the front page. When you click it, you just see the contents of the symlink file ("IntxLNKBGMApp/BGMApp/LICENSE").

Also, this makes some sites list the license as "other": https://awesomeopensource.com/project/kyleneideck/BackgroundMusic

**Comment by kyleneideck** [comment]
Just duplicating the LICENSE file is probably a perfectly good fix for this. That is, replacing the symlink with a copy of the file.

**Comment by nodeful** [comment]
Hey Kyle,
We have spoken before, I've based [eqMac's](https://github.com/bitgapp/eqMac) driver on your BGMDriver source code.
Does the GPLv2 License apply to the driver as well? Currently the LICENSE file lives in the BGMApp folder, which led me to believe it only applies to the App itself, not the driver.
I am planning to release some paid features to eqMac very soon to support the development and would like to know how do you see the licensing work with this. 
The paid features will be on a private form of the OS repo, so closed source, although any changes to the Driver I would still publish to the OS repo. 
Maybe we need to take this conversation offline, I have your WhatsApp from years ago. Let me know :) Awesome work as always btw

**Comment by kyleneideck** [comment]
@nodeful Yeah, the whole thing is GPLv2 (or any later version). But I think you'd be allowed to "aggregate" a GPL driver with your proprietary version of eqMac as long as you release the driver under the GPL: <https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.en.html#MereAggregation>. So it sounds like that's probably fine.

That said, and this doesn't have anything to do with Background Music, but the other contributors to eqMac would still hold the copyright for the code they contributed, unless they assigned it to you or signed a CLA. So if any of that code would be in the proprietary app, you'll probably need to get permission from them.

Feel free to let me know if you've got any other questions about it.

**Comment by Arsenic-ATG** [comment]
sorry if it sound silly but is there a specific reason for the licence file to be a symlink at the first place ? 😕

**Comment by kyleneideck** [comment]
@Arsenic-ATG The app shows the license text in the About Background Music window, so we needed a copy of the license file in the Xcode project dir. I guess I made `/LICENSE` a symlink to that file, rather than a duplicate, just in case we needed to change one and forgot to change the other. Or something like that.

But I doubt that's actually worth worrying about, so I think we should just have two copies of the file.
